content/events/hacktoberfest.adoc

```diff
@@ -230,24 +222,20 @@ link:https://issues.jenkins.io/issues/?jql=labels%20%3D%20newbie-friendly%20AND%
 
 |=========================================================
 
-
 === Experienced developers
 
-If you are an established developer and want to create something new,
-please don't let yourself to be blocked by the suggested topics!
+If you are an established developer and want to create something new, please don't let yourself to be blocked by the suggested topics!
```

Comment:
```suggestion
If you are an established developer and want to create something new, feel free to explore beyond the suggested topics!
```
suggestion to change the text so that it makes a bit more sense, while still giving the same encouragement